src/client.js

```diff
@@ -170,6 +170,38 @@ class CommandoClient extends discord.Client {
 		return undefined;
 	}
 
+	/**
+	 * Sets the setting provider to use, and initialises it once the client is ready
+	 * @param {SettingProvider} provider Provider to use
+	 * @return {void}
+	 */
+	setBetterSQLite3Provider(provider) {
```

Comment:
There's no point to having a separate method to set this provider. `setProvider` is made to be provider-agnostic already.

Reply:
While that is entirely true in practice, wouldn't it be unprofitable to be using the async/await approach of `setProvider` for a synchronous connection such as made by better-sqlite3? That is the reason why I wrote my own method, to write is as sync.

I mean sure I can remove it of course no problem, but I'd first like to hear some feedback on that reasoning.